app/controllers/api/v1/ontologies_controller.rb

```diff
@@ -22,4 +23,8 @@ def version
     finder = OntologyVersionFinder.new(params[:reference], resource)
     @version ||= finder.find if params[:reference]
   end
+
+  def set_format
```

Comment:
Should be moved to a Controller concern when porting to Rails 4. If there is an elegant solution for sharing this with other Controllers, that works with Rails 3, I'm all ears.